Fix dns config in cloudformation template

noamkush opened the pull request:
While trying to use the cloudformation template I noticed a small issue that can be avoided with the DNS configuration. The zone id for the alias target (the load balancer) should be taken from the load balancer, it's not a template parameter.

See relevant [doc](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-route53-aliastarget-1.html#cfn-route53-aliastarget-hostedzoneid).

juanpicado commented:
> While trying to use the cloudformation template I noticed a small issue that can be avoided with the DNS configuration. The zone id for the alias target (the load balancer) should be taken from the load balancer, it's not a template parameter.
> 
> See relevant [doc](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-route53-aliastarget-1.html#cfn-route53-aliastarget-hostedzoneid).

Since I'm not particular skilled (maybe some @verdaccio/collaborators  is) on AWS CDN maybe you could elaborate a bit more, the issue and why that solution on this PR.

noamkush commented:
The configuration I modified defines the route53 (dns) record which points to the load balancer. It's configured as an alias so you need to give details about the alias target. Since the target here is a load balancer which hasn't yet been created at all, there's no way the template user would know which zone it would be created in. What I added is automatic configuration for this detail, as the docs specify:

> ELB load balancer
    Specify the value of the hosted zone ID for the load balancer. Use the following methods to get the hosted zone ID:
    ...
        CloudFormation Fn::GetAtt intrinsic function: Use the Fn::GetAtt intrinsic function to get the applicable value:
            Classic Load Balancers: Get CanonicalHostedZoneNameID.
            **Application and Network Load Balancers: Get CanonicalHostedZoneID.**

This enabled removing this parameter so it should be easier to use the template.
